Outline Terraform module

tdemint10 opened the pull request:
Basic outline of the module is created with simple setup and documentation.

tdemint10 committed:
Outline Terraform module

Basic outline of the module is created with simple setup and
documentation.

lamarmeigs commented on main.tf:
```diff
@@ -0,0 +1,15 @@
+terraform {
+  required_version = ">= 1.8.0"
```
🔥

lamarmeigs reviewed (approved):
A thought: would it be worth configuring pre-commit hooks from the beginning? (I hadn't done that previously because I was committing directly to main, but for a self-contained PR, maybe it's appropriate?)

tdemint10 committed:
Terraform pre-commit hooks

The pre-commit hooks will ensure conventional formatting and catch more
egregious errors before commits are finalized. They'll also ensure that
the README stays up-to-date.
